documentation/disaster-recovery.md

```diff
@@ -0,0 +1,59 @@
+# Disaster recovery
```

Comment:
We should add this link somewhere: https://docs.cloud.service.gov.uk/deploying_services/postgresql/#restoring-a-postgresql-service-from-a-point-in-time